Formula/clang-format-8.rb

```diff
@@ -0,0 +1,50 @@
+class ClangFormat8 < Formula
+  desc "Formatting tools for C, C++, Obj-C, Java, JavaScript, TypeScript"
+  homepage "https://clang.llvm.org/docs/ClangFormat.html"
+  url "https://github.com/llvm/llvm-project/releases/download/llvmorg-8.0.1/llvm-8.0.1.src.tar.xz"
+  sha256 "44787a6d02f7140f145e2250d56c9f849334e11f9ae379827510ed72f12b75e7"
+  license "Apache-2.0"
+
+  depends_on "cmake" => :build
+  depends_on "ninja" => :build
+
+  uses_from_macos "libxml2"
+  uses_from_macos "ncurses"
+  uses_from_macos "zlib"
+
+  resource "clang" do
+    url "https://github.com/llvm/llvm-project/releases/download/llvmorg-8.0.1/cfe-8.0.1.src.tar.xz"
+    sha256 "70effd69f7a8ab249f66b0a68aba8b08af52aa2ab710dfb8a0fba102685b1646"
+  end
+
+  resource "libcxx" do
+    url "https://github.com/llvm/llvm-project/releases/download/llvmorg-8.0.1/libcxx-8.0.1.src.tar.xz"
+    sha256 "7f0652c86a0307a250b5741ab6e82bb10766fb6f2b5a5602a63f30337e629b78"
+  end
+
+  def install
+    (buildpath/"projects/libcxx").install resource("libcxx")
+    (buildpath/"tools/clang").install resource("clang")
+
+    mkdir buildpath/"build" do
+      args = std_cmake_args
+      args << "-DLLVM_ENABLE_LIBCXX=ON"
+      args << ".."
+      system "cmake", "-G", "Ninja", *args
+      system "ninja", "clang-format"
+    end
+
+    bin.install buildpath/"build/bin/clang-format" => "clang-format-8"
+    bin.install buildpath/"tools/clang/tools/clang-format/git-clang-format" => "git-clang-format-8"
```

Comment:
Am I correct in assuming that these binaries don't conflict with anything installed by the main `clang-format` formula?

Reply:
Yes, it doesn't conflict. That's why I change formula `ClangFormatAT8` to `ClangFormat8`. Is there a batter way how allow install binaries as stand alone formula? I means `ClangFormatAT8` needs keg-only and then it doesn't install into `/usr/local`.

Is there is no other way tell me if formula realy needs to be `ClangFormatAT8`?